app/src/components/docGen.js

```diff
@@ -1,34 +1,35 @@
-const log = require('npmlog');
 const carbone = require('carbone');
+const fs = require('fs');
+const log = require('npmlog');
 const stream = require('stream');
 const tmp = require('tmp');
-const fs = require('fs');
 const utils = require('./utils');
 
+tmp.setGracefulCleanup();
+
 const docGen = {
   /** Generate a file using carbone to merge data into the supplied document template
    *  @param {object} body The request body
    *  @param {object} response The server response to write the generated file to
    */
-  generateDocument: async (body, response) => {
-    tmp.setGracefulCleanup();
-    let tmpFile = undefined;
+  generateDocument: (body, response) => {
+    let tmpFile;
     try {
-      tmpFile = tmp.fileSync();
+      const inboundFileExtension = utils.getFileExtension(body.template.filename);
```

Comment:
This is just the Docker container stuff, so this file is being changed by me in a differnt PR (which we'll coordinate, discussed with @jujaga) for file extension stuff

Reply:
But how do we know that it actually picks up and converts to pdf?  Since this method just takes the body, we could add some logic to take an additional undocumented field that we could use from Postman.  Or we could just hardcode the convertTo to PDF and for this PR, everything gets converted to PDF?  I'd like to know that the dockerimage is fully functional

Reply:
Right, could throw something temporary into this PR for testing I suppose. @jujaga ?

Reply:
When I tested this locally, I temporarily hardcoded Line 32 to `convertTo: 'pdf'` to force a PDF execution conversion. I was able to verify it was working with a docx and a odt file. I could do that again and have it actually be a part of this PR, but I suspect it'd block actual functionality of doing no-conversion flows on live.

Reply:
@parc-jason I just pushed up a temporary commit with Line 32 hard coded in. Please independently verify and let me know that it works for you. DO NOT merge this commit in with that hard-code in place - I will need to roll back that commit later before this PR gets merged.

Reply:
verified that file can be converted to PDF.
Rollback that commit. I will approve but not merge.